app/models/event/camp.rb

```diff
@@ -64,4 +64,8 @@ class Event::Camp < Event
 
   belongs_to :kind, class_name: 'Event::Camp::Kind'
 
+  scope :in_next, -> (duration) {
+    timespan = Time.zone.now.midnight + duration
```

Comment:
```suggestion
    timespan = duration.from_now.to_date
```

Wenn hier eine negative `duration` übergeben wird, wird es wie `duration.abs.ago` gewertet. Durch die Benennung vermute ich, dass es eine positive Zahl/Dauer sein soll.